DevIncubator-Autopark/VehicleHelper.cs

```diff
@@ -35,5 +35,19 @@ public static Vehicle GetVehicleWithMinMileage(IReadOnlyList<Vehicle> vehicles)
             };
             return minMileageVehicle;
         }
+
+        public static void PrintEqualsVehicles(IReadOnlyList<Vehicle> vehicles)
+        {
+            for (int i = 0; i < vehicles.Count; i++)
+            {
+                for (int j = i + 1; j < vehicles.Count; j++)
```

Comment:
IndexOutOfRange is thrown here

Reply:
> IndexOutOfRange is thrown here 

Why?

Reply:
> 

No exceptions. This method works well

Reply:
Yeah, you are right, sorry

Reply:
> 

You can write in first loop `vehicles.Count - 1` to avoid 1 useless loop iteration ^_^ i think it's not a bug or error, but... ;)